### PR TITLE
[YUNIKORN-1412] fix render issue in Scheduler cache removal design doc.

### PR DESCRIPTION
**Pull request by KatLantyss**
Fix render issue in Scheduler cache removal design doc.

### CODE REVIEW

File: docs/design/cache_removal.md [hunk 1]
```diff
@@ -32,10 +32,14 @@ It describes the changes planned based on the analysis that was done of the curr
 
 ## Goals
 The goal is to provide the same functionality before and after the change.
-- unit tests before and after the merge must all pass.
-- Smoke tests defined in the core should all pass without major changes <sup id="s1">[definition](#f1)</sup>.
+- Unit tests before and after the merge must all pass.
+- Smoke tests defined in the core should all pass without major change.
```
Comment (craigcondit) [inline docs/design/cache_removal.md:36]: 'change' should be 'changes' here.

File: docs/design/cache_removal.md [hunk 2]
```diff
@@ -47,10 +51,11 @@ This short period can have an impact on the scheduling decisions.
 One of which is logged as [YUNIKORN-169](https://issues.apache.org/jira/browse/YUNIKORN-169).
 
 A further point is the complexity that the two structure brings to the code.
-A distinct set of messages to communicate between the scheduler and the cache.
-A one on one mapping between the scheduler and cache objects shows that the distinction is probably more artificial than required.
+A distinct set of messages to communicate between the scheduler and the cache. A one on one mapping between the scheduler and cache objects shows that the distinction is probably more artificial than required.
```
Comment (craigcondit) [inline docs/design/cache_removal.md:54]: one on one should be hyphenated: one-on-one

File: package.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "@docusaurus/core": "2.1.0",
     "@docusaurus/preset-classic": "2.1.0",
-    "@docusaurus/theme-search-algolia": "2.1.0",
+    "@docusaurus/theme-search-algolia": "^2.2.0",
```
Comment (craigcondit) [inline package.json:15]: Please remove this from the PR.

**Review by craigcondit — CHANGES_REQUESTED** [review]
A few minor changes.

**Review by wilfred-s — APPROVED** [review]
LGTM, nits from @craigcondit have been resolved

**Pull request closed without merging**